core/src/main/scala/akka/persistence/cassandra/EventsByTagMigration.scala

```diff
@@ -107,22 +107,31 @@ object EventsByTagMigration {
 
 }
 
-class EventsByTagMigration(system: ActorSystem)
+/**
+  *
+  * @param journalNamespace The config namespace where the journal is configured, default is `cassandra-journal`
+  * @param readJournalNamespace The config namespace where the query-journal is configured, default is the same
+  *                             as `CassandraReadJournal.Identifier`
+  */
+class EventsByTagMigration(system: ActorSystem,
+                           journalNamespace: String = "cassandra-journal",
```

Comment:
I don't know that I'd pass in a default param like this. If I had to do it, I'd make it a val somewhere so you could at least call something like `CassandraReadJournal.Identifier`. Seems arbitrary and potentially brittle.

Reply:
Maybe something to improve upon when we rework the configuration at some point (#81)

Since this is just a migration tool I don't think it matters much if it is brittle, you pretty much run it once and then you have migrated.